.docs/user-guide/troubleshooting.md

```diff
@@ -0,0 +1,241 @@
+# Troubleshooting
+
+It's not doing what I expected...
+
+---
+
+## Configuration Validation
+REX-Ray configuration will usually be set in the yaml file located at `/etc/rexray/config.yml`
+
+If you are having issues, or anytime you edit your configuration you may wish to validate the result by cut and paste to an online validator such as this one: [http://yaml-online-parser.appspot.com/](http://yaml-online-parser.appspot.com/)
+
+If REX-Ray is running as a service, the service needs to be restarted to pick up the change.
+
+`sudo rexray service restart`
+
+The configuration file is the conventional method to set configuration, but this can be supplemented or replaced by environment variables and command line option flags. For example, when running a centralized REX-Ray controller service in a Docker container, you would likely use an environment variables configuration. 
+
+If configuration is set in in multiple locations, the precedence order is:
+
+Command line overrides -> environment overrides -> config file
+
+The command `rexray env` can be used to dump the current _effective_ configuration - in an environment variable form, regardless of whether environment variables were used to set the configuration.
+
+### Ultra simplified configuration 
+A minimal complexity configuration can be used as an experiment to eliminate issues external to configuration.
+
+1. Shut down the REX-Ray service: `sudo rexray service stop`
+2. Save your original config: `sudo mv /etc/rexray/config.yml /etc/rexray/config.$(date +%Y-%m-%d-%T).yml`
+3. Create a new minimal config `vi \etx\rexray\config.yml`
+
+This is a minimal configuration for **AWS EBS** volumes. Other storage drivers are similar and will mostly vary along the lines of authorization credentials, and possibly storage API address identification.
+
+```
+rexray:
+  loglevel:warn
+libstorage:
+  service: ebs
+ebs:
+  accessKey: YOUR-ACCESS-KEY
+  secretKey: your-secret-key
+  region:us-west-2
+```
+
+This will parse the config and emit it in environment variable form 
+
+`rexray env`
+
+Next test that your REX-Ray installation works in interactive (non service) form:
+
+`rexray volume ls –s ebs`
+
+
+if this works, move on to operating as a service:
+
+`sudo rexray service start`
+
+Look at the log (`/var/log/rexray/rexray.log`) which should show the version and no errors. 
+
+A few other minimal REX-Ray configuration examples for other platforms:
+
+**ScaleIO** (host_ip varies)
+
+```
+rexray:
+  Loglevel:warn
+libstorage:
+  service: scaleio
+scaleio:
+  Endpoint: https://host_ip/api
```

Comment:
Hi @cantbewong,

While it will work, please keep all keys consistent with each other and rename this `endpoint` (casing).